### PR TITLE
correction of testdata json5

### PR DESCRIPTION
**Pull request by Krisselchen**
put files from janette in correct json5 format. Inserting of data should work now.

### CODE REVIEW

File: evalData/janette_10.json5 [hunk 1]
```diff
@@ -1,50 +1,48 @@
...
-  "status": "DONE",
-  "description": "
-After fixing the problems with the fallback logic, we re-run the experiment. 
-We observe that Gizeh is outperforming the Spirit when looking at the time to play (latency). The decrease is xxx. 
-  ",
-  "date_creation": "2016-07-05",
-  "date_last_updated": "2016-07-15",
-  "tags": [
+  status: "DONE",
+  description: "After fixing the problems with the fallback logic, we re-run the experiment.
```
Comment (henderkes) [inline evalData/janette_10.json5:30]: multi-line strings have to be escaped

File: evalData/janette_12.json5 [hunk 2]
```diff
@@ -1,37 +1,35 @@
...
+  team: {
+    team_name: "data-science-listener"
   },
-  "status": "IN_PROGRESS",
-  "description": "
-We analyze the churn rate of new users and how churn vs. retained users differ. Questions we are trying to answer:
+  status: "IN_PROGRESS",
+  description: "We analyze the churn rate of new users and how churn vs. retained users differ. Questions we are trying to answer:
 How high is our churn rate on different platforms, in different countries, etc.?
 Who are the users that churn on day-0, week-0 or month-0?
```
Comment (DubbleClick) [inline evalData/janette_12.json5]: see above

**Review by henderkes — COMMENTED** [review]
there are still some unescaped new lines

why did you remove the quotation marks of the property?

**Comment by henderkes** [comment]
I guess it's safe to say that this won't ever land in master as they didn't want the manifests online

**Pull request closed without merging**